DSMAJCEexamples/src/main/java/DSMAJCE.java

```diff
@@ -0,0 +1,119 @@
+import com.fortanix.sdkms.jce.provider.SdkmsJCE;
+import com.fortanix.sdkms.jce.provider.keys.asym.rsa.RSAPrivateKeyImpl;
+import com.fortanix.sdkms.jce.provider.keys.asym.rsa.RSAPublicKeyImpl;
+import com.fortanix.sdkms.jce.provider.keys.sym.aes.SdkmsAESKey;
+import com.fortanix.sdkms.jce.provider.service.SdkmsKeyService;
+import com.fortanix.sdkms.v1.model.KeyObject;
+import com.fortanix.sdkms.v1.model.SobjectDescriptor;
+import static com.fortanix.sdkms.jce.provider.constants.ProviderConstants.*;
+
+import javax.crypto.*;
+import java.io.UnsupportedEncodingException;
+import java.nio.ByteBuffer;
+import java.security.*;
+import java.security.spec.MGF1ParameterSpec;
+import java.security.spec.PSSParameterSpec;
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+
+public class DSMAJCE {
+    // replace with dsm url for env you are pointing to.
+    private static String FORTANIX_API_ENDPOINT = "<Endpoint URL>";
+
+    // replace with your api-key
+    private static String FORTANIX_API_KEY = "<API Key>";
+    // replace with your app uuid. Only required for cert-based auth
+    private static String FORTANIX_APP_UUID = "<APP UUID>";
+
```

Comment:
Rather than address key id and key name with static references below in the code, we should create variables here that will refer to the key id and name.  So, aes_keyname, aes_keyid, rsa_keyname, rsa_keyid.....etc.  Then refer to them in the code.